janusgraph-hbase-parent/janusgraph-hbase-core/src/test/java/org/janusgraph/HBaseStorageSetup.java

```diff
@@ -48,7 +48,9 @@
 
     public static final String HBASE_PARENT_DIR_PROP = "test.hbase.parentdir";
 
-    private static final Pattern HBASE_SUPPORTED_VERSION_PATTERN = Pattern.compile("^((0\\.9[468])|(1\\.[01]))\\..*");
+    private static final Pattern HBASE_SUPPORTED_VERSION_PATTERN = Pattern.compile("^((0\\.9[8])|(1\\.[012]))\\..*");
```

Comment:
This is a test file.

I tested with the different HBase versions, and looked at the test.log files.  They are matched and mapped fine.

mvn test -pl :janusgraph-hbase-10   // default with hbase 1.2.4
mvn test -Dhbase100.core.version=1.1.8 -pl :janusgraph-hbase-10
mvn test -Dhbase100.core.version=1.0.3 -pl :janusgraph-hbase-10
mvn test -pl :janusgraph-hbase-098

2017-02-06 18:21:47.366 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 0.98.2-hadoop2: ../janusgraph-hbase-098/bin/

2017-02-06 13:45:33.876 [main] DEBUG janusgraph.HBaseStorageSetup: Built conf path for HBase version 1.2.4: ../janusgraph-hbase-10/bin/

2017-02-06 15:41:22.594 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 1.1.8: ../janusgraph-hbase-10/bin/

2017-02-06 16:43:30.543 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 1.0.3: ../janusgraph-hbase-10/bin/